ci/citest/acceptance-test/multibox/10.0.100.17-vdc-executor-qemu-ovs/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export ENV_ROOTDIR="$(cd "$(dirname $(readlink -f "$0"))/.." && pwd -P)"
+export NODE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TMP_ROOT="${NODE_DIR}/tmp_root"
+
+. "${ENV_ROOTDIR}/config.source"
+. "${NODE_DIR}/vmspec.conf"
+. "${ENV_ROOTDIR}/ind-steps/common.source"
+
+IND_STEPS=(
+    "box"
+    "ssh"
+    "hosts"
+    "disable-firewalld"
+    "epel"
+    "openvswitch"
+    "qemu"
+    "kvm"
```

Comment:
Just my 2 cents, but qemu can run lots of things that we might implement in the future at some point (like xen). So it might make sense to leave them separated especially since it's already done that way...